### ISSUE TITLE
Editor state getting reverted using header and console logs "Block «header» skipped because saved data is invalid".

### ISSUE THREAD

**Issue by HarkiratShopism1** [codex-team/editor.js#2767]
### Describe the bug. 
I am using editorJs in my react application as a form component and when trying to add a Paragraph or Header I am getting following error in the console "Block «header» skipped because saved data is invalid" for header and "Block «paragraph» skipped because saved data is invalid" for Paragraph also the editor state gets reverted back to blocks -> 0 and I am not able to get the header or paragraph on the next line. Following is the sample react code attached how I have initialized editorJs in my application.

```
import React, { useEffect, useRef } from 'react';
import EditorJS from '@editorjs/editorjs';
import Header from '@editorjs/header';
import List from '@editorjs/list';
import Paragraph from '@editorjs/paragraph';

const Editor = ({ data, onChange }) => {
    const ejInstance = useRef(null);
    const editorContainer = useRef(null);

    useEffect(() => {
        if (!ejInstance.current) {
            ejInstance.current = new EditorJS({
                holder: editorContainer.current,
                tools: {
                    header: Header,
                    list: List,
                    paragraph: {
                        class: Paragraph,
                        inlineToolbar: true,
                    },
                },
                data: data,
                onChange: async () => {
                    const content = await ejInstance.current.save();
                    onChange(content);
                },
            });
        }

        return () => {
            if (ejInstance.current && typeof ejInstance.current.destroy === 'function') {
                ejInstance.current.destroy();
                ejInstance.current = null;
            }
        };
    }, [data, onChange]);

    return <div ref={editorContainer}></div>;
};

export default Editor;
```

### Screenshots: 
![image](https://github.com/codex-team/editor.js/assets/132342163/d866e80b-8e73-4323-8f09-bf127cc71660)

**Comment by neSpecc** [comment]
Do you mean that empty headers and paragraph are skipped?

**Comment by HarkiratShopism1** [comment]
Yes exactly.

**Comment by HarkiratShopism1** [comment]
> Do you mean that empty headers and paragraph are skipped?

Hi @neSpecc do you have any solution or fix for this issue?

**Comment by neSpecc** [comment]
Try using header 2.8.7

**Comment by HarkiratShopism1** [comment]
I have tried multiple things but issue not getting resolved
Removed React strict mode 
Installed 2.8.7 headers 

Nothing worked
Can anyone help me with this

**Comment by ansonurpants** [comment]
hi im having the same issue, has anyone found the solution?

**Comment by neSpecc** [comment]
Please, clarify the problem. I'm not sure I've got it correctly. Right now Editor.js skips blank blocks (those which specified the `validate()` method for that). 


Do you want to preserve blank lines? 

And what does it mean `Editor state getting reverted` ?

**Comment by ansonurpants** [comment]
yeah the problem is the validate function runs when first creating the heading line so it skips the block because the line will always be blank like with the paragraph block. once the heading block is skipped, the heading styling no longer shows up so I'm the editor it just shows up like paragraph text even tho in the Json it's saved as a heading block.

**Comment by neSpecc** [comment]
> once the heading block is skipped, the heading styling no longer shows up

why? block is not removed. It's just skipped in output JSON

**Comment by ansonurpants** [comment]
hmm you're right, it might have something to do with using CDN instead of
npm?
